src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -253,9 +256,9 @@ public DirectoryInfo ComputeProjectBaseDir(IEnumerable<DirectoryInfo> projectPat
                 logger.LogDebug(Resources.MSG_UsingWorkingDirectoryAsProjectBaseDir, workingDirectory.FullName);
                 return workingDirectory;
             }
-            else if (PathHelper.BestCommonPrefix(projectPaths) is { } commonPrefix)
+            else if (PathHelper.BestCommonPrefix(projectPaths, pathComparer) is { } commonPrefix)
             {
-                logger.LogDebug(Resources.MSG_UsingLongestCommonBaseDir, commonPrefix.FullName);
+                logger.LogDebug(Resources.MSG_UsingLongestCommonBaseDir, commonPrefix.FullName, Environment.NewLine + string.Join($"{Environment.NewLine}", projectPaths.Select(x => x.FullName)));
```

Comment:
the coverage seems to have a bug here. A branch appears as not tested but there are no conditionals.